MAUI/ImageEditor/getting-started.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: Getting Started with .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about getting started with Syncfusion .NET MAUI ImageEditor(SfImageEditor) control
+platform: maui
+control: SfImageEditor
+documentation: ug
+---
+
+# Getting Started with .NET MAUI ImageEditor (SfImageEditor)
+

```

Comment:
This section explains the steps required to create and load an image to the image editor control.